packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,57 @@
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #f4f4f4);
+    border-bottom: var(--terra-action-header-border-bottom, 1px solid #dedfe0);
+    border-left: var(--terra-action-header-border-left);
+    border-right: var(--terra-action-header-border-right);
+    border-top: var(--terra-action-header-border-top);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
+    
+    .flex-collapse {
+      align-items: center;
+      display: flex;
+      flex: 0 0 auto;
+      height: 100%;
+      max-width: 40%;
+      position: relative;
+    }
+  }
+
+  .flex-fill {
+    align-self: center;
+    flex: 1 1 auto;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-end {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    position: relative;
+  }
+
+  .title-container {
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .title {
+    font-size: var(--terra-action-header-font-size, 1.286rem);
+    font-weight: normal;
+    height: 100%;
+    margin-bottom: 0;
+    margin-top: 0;
+    width: 100%;
+    word-break: normal;
```

Comment:
Found an issue when a really long word with no spaces/breaks is used for the title, it will overflow the layout. Possible with translation when action-header is used in a small container and also includes a collapsible-menu. 

Recommend changing to `word-break: break-all;` and also adding:
```
-webkit-hyphens: auto;
    -ms-hyphens: auto;
        hyphens: auto;
```